src/main/scala/is/hail/methods/MendelErrors.scala

```diff
@@ -137,25 +138,25 @@ case class MendelErrors(hc: HailContext, vSig: Type, trios: IndexedSeq[CompleteT
 
   def mendelKT(): Table = {
     val signature = TStruct(
-      "fid" -> TString(),
-      "s" -> TString(),
+      "fam_id" -> TString(),
+      "id" -> TString(),
```

Comment:
why this change?

I think when we have general keys, we should just copy over the dataset col key field.